cmds/exp/acpi2coreboot/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// build coreboot C code from tables
```

Comment:
nit: can you add the name here?

// acpi2coreboot builds coreboot C code from ACPI tables